js/browser.js

```diff
@@ -312,7 +312,7 @@ class Browser {
      * Render browse display as SVG
      * @returns {string}
      */
-    toSVG() {
+    async toSVG() {
```

Comment:
this should not be async; it wasn't in the original code & we don't want it to be.  browser state should not change while an export is happening

Reply:
justified b/c ROI features must be resolved from cache